static/app/views/explore/multiQueryMode/queryConstructors/groupBy.tsx

```diff
@@ -4,15 +4,26 @@ import styled from '@emotion/styled';
 import {CompactSelect, type SelectOption} from 'sentry/components/compactSelect';
 import {t} from 'sentry/locale';
 import {useSpanTags} from 'sentry/views/explore/contexts/spanTagsContext';
+import {
+  type ReadableExploreQueryParts,
+  useUpdateQueryAtIndex,
+} from 'sentry/views/explore/multiQueryMode/locationUtils';
 import {
   Section,
   SectionHeader,
   SectionLabel,
 } from 'sentry/views/explore/multiQueryMode/queryConstructors/styles';
 
-export function GroupBySection() {
+type Props = {
+  index: number;
+  query: ReadableExploreQueryParts;
+};
+
+export function GroupBySection({query, index}: Props) {
   const tags = useSpanTags();
 
+  const updateGroupBys = useUpdateQueryAtIndex();
```

Comment:
Any reason why these hooks dont take the index as an argument directly? Instead of having to wrap it again further down?

Reply:
No reason! I think I might change the hooks to take the index arg here since it reads better with the name of the hook